rasa_core/evaluate.py

```diff
@@ -204,7 +204,7 @@ class EndToEndUserUtterance(UserUttered):
     Mostly used to print the full end-to-end user message in the
     `failed_stories.md` output file."""
 
-    def as_story_string(self):
+    def as_story_string(self, e2e=True):
```

Comment:
Is this kwarg necessary?

Reply:
I believe functionally it isn't but it preserves the signature of the parent method

Reply:
Yes, exactly.